spec/features/admin/orders_spec.rb

```diff
@@ -72,7 +72,7 @@ def new_order_with_distribution(distributor, order_cycle)
     expect(page).not_to have_selector '#errorExplanation'
     expect(page).to have_content 'ADD PRODUCT'
     targetted_select2_search @product.name, from: '#add_variant_id', dropdown_css: '.select2-drop'
-    click_link 'Add'
+    find('button.add_variant').click
```

Comment:
Why not `click_button "Add"`?

Reply:
it's a plus sign (image) 
![screenshot 2019-02-07 at 13 39 33](https://user-images.githubusercontent.com/1640378/52415034-de772500-2add-11e9-83c9-5a43c1803759.png)
now: